level2/rds.tf

```diff
@@ -0,0 +1,23 @@
+data "aws_secretsmanager_secret" "rds_password" {
+    name = "rds_password"
+}
+
+data "aws_secretsmanager_secret_version" "rds_password" {
+    secret_id = data.aws_secretsmanager_secret.rds_password.id
+}
+
+# translate the JSON text into a terraform object and query using the standard terraform secret. 
+# A way to declare variables, similar to local env variables. 
+locals {
+    rds_password = jsondecode(data.aws_secretsmanager_secret_version.rds_password.secret_string)["password"]
+}
+
+module "rds" {
+  source = "../modules/rds"
+
+  subnet_ids            = data.terraform_remote_state.level1.outputs.private_subnet_id
+  env_code              = var.env_code
+  rds_password          = local.rds_password
+  vpc_id                = data.terraform_remote_state.level1.outputs.vpc_id
+  source_security_group = module.asg.security_group_id 
+}
```

Comment:
Could you please, add a new line at the end of file.

Reply:
Added a new line.